pkg/cmd/pr/status/status.go

```diff
@@ -179,20 +179,20 @@ func prSelectorForCurrentBranch(baseRepo ghrepo.Interface, prHeadRef string, rem
 	return
 }
 
-func printPrs(w io.Writer, totalCount int, prs ...api.PullRequest) {
+func printPrs(cs *iostreams.ColorScheme, w io.Writer, totalCount int, prs ...api.PullRequest) {
```

Comment:
Another case where just passing in `*iostreams.IOStreams` could be used, if we decide to go that direction.